cluster/engine.go

```diff
@@ -1164,7 +1164,7 @@ func (e *Engine) handler(msg events.Message) error {
 			// These events refer to images so there's no need to update
 			// containers.
 			e.RefreshImages()
-		case "die", "kill", "oom", "pause", "start", "stop", "unpause", "rename":
+		case "die", "kill", "oom", "pause", "start", "stop", "unpause", "rename", "update":
```

Comment:
you can following:
1.modify the code in the same branch locally,
2.`git add`
3. when `git commit`, using `git commit -s --amend -m "xxx" `
4.`git push -f`